x-pack/plugins/security_solution/server/endpoint/routes/actions/list.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ENDPOINTS_ACTIONS_LOG_ROUTE } from '../../../../common/endpoint/constants';
+import { AgentsActionsLogRequestSchema } from '../../../../common/endpoint/schema/actions';
+import { actionListHandler } from './list_handler';
+
+import type { SecuritySolutionPluginRouter } from '../../../types';
+import type { EndpointAppContext } from '../../types';
+import { withEndpointAuthz } from '../with_endpoint_authz';
+
+/**
+ * Registers the endpoint activity_log route
+ */
+export function registerActionListRoutes(
+  router: SecuritySolutionPluginRouter,
+  endpointContext: EndpointAppContext
+) {
+  router.get(
+    {
+      path: ENDPOINTS_ACTIONS_LOG_ROUTE,
+      validate: AgentsActionsLogRequestSchema,
+      options: { authRequired: true, tags: ['access:securitySolution'] },
+    },
+    withEndpointAuthz(
+      { all: ['canIsolateHost'] },
```

Comment:
I might be out-of-the-loop on this, but what does 'canIsolateHost' permission otherwise map to? Is it so common that we might want to rename what that permission entails internally? This appears to be a read-only API, but using a modify-sounding permission.

If we expect levels to actually be granular, and there might be other security permissions in use, then could we let something more permissive here? Seeing a log that includes (user X isolated Z) seems ok to be a lower bar, and a lower permission OK

Reply:
Changed it to use `canAccessEndpointManagement`